CollisionOperator/CollisionProcessors/neutronCEimp_class.f90

```diff
@@ -572,19 +573,30 @@ subroutine split(self, p, thisCycle, maxWgt)
     class(particle), intent(inout)        :: p
     class(particleDungeon), intent(inout) :: thisCycle
     real(defReal), intent(in)             :: maxWgt
-    integer(shortInt)                     :: mult, i
+    integer(shortInt)                     :: mult, i, splitCount
 
     ! This value must be at least 2
     mult = ceiling(p % w/maxWgt)
 
+    ! Limit maximum split
+    if (mult > maxSplit - p % splitCount + 1) then
```

Comment:
```suggestion
    if (mult + p % splitCount > maxSplit) then
```
Maybe this condition will be a bit more readable?
I had a bit of trouble to work out the intention behind this if statement. It does limit number of 2ndary particles created by splitting so they do not exceed `maxSplit`, right? Or did I misunderstood something?

Reply:
Yes, sure! That was the intention, I might have made it more convoluted than it needed to be! Thanks